browser-api/src/main/java/com/duckduckgo/app/tabs/model/TabSwitcherData.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.tabs.model
+
+data class TabSwitcherData(
+    val userState: UserState,
+    val wasAnnouncementDismissed: Boolean,
+    val announcementDisplayCount: Int,
+) {
+    enum class UserState {
+        NEW,
+        RETURNING,
```

Comment:
Maybe rename this to `EXISTING`.

Reply:
Done in [81e783b](https://github.com/duckduckgo/Android/pull/4743/commits/81e783b658c4c21035e19a9d4f742c5e60596e1e).